src/DSE.Open.Observations/SentenceFrequencyObservation.cs

```diff
@@ -43,7 +43,7 @@ public static SentenceFrequencyObservation Create(
     [SkipLocalsInit]
     protected override ulong GetDiscriminatorId()
     {
-        var chars = SentenceId.ToCharSequence().AsSpan();
+        var chars = SentenceId.ToString();
```

Comment:
Not sure why we don't just return `(ulong)SentenceId` or, perhaps, `XxHash3.HashToUInt64(BitConverter.GetBytes((ulong)SentenceId))`?